src/Assets.php

```diff
@@ -126,6 +126,8 @@ public static function get_wc_block_data( $settings ) {
 				'productCount'                => array_sum( (array) $product_counts ),
 				'attributes'                  => array_values( wc_get_attribute_taxonomies() ),
 				'wcBlocksAssetUrl'            => plugins_url( 'assets/', __DIR__ ),
+				'shippingCountries'           => wp_json_encode( WC()->countries->get_shipping_countries() ),
+				'allowedCountries'            => wp_json_encode( WC()->countries->get_allowed_countries() ),
```

Comment:
does this need to be encoded here? I think it's already handled by https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/src/Assets/AssetDataRegistry.php#L219.  No need to encode twice?

Reply:
Right! Removed in d6eba824dddff3cc3a1785af064b23de79104abd.